asgiref/conformance.py

```diff
@@ -29,22 +29,10 @@ class ConformanceTestCase(unittest.TestCase):
     capacity_limit = None
 
     @classmethod
-    def raiseSkip(cls, message):
-        """
-        Picks the right skip class based on what version of unittest we're using.
-        """
-        try:
-            import unittest2
-            SkipTest = unittest2.SkipTest
-        except ImportError:
-            SkipTest = unittest.SkipTest
-        raise SkipTest(message)
-
-    @classmethod
     def setUpClass(cls):
         # Don't let this actual class run, it's abstract
         if cls is ConformanceTestCase:
-            cls.raiseSkip("Skipping base class tests")
+            raise unittest.SkipTest("Skipping base class tests")
```

Comment:
I'm afraid I don't understand; could you please clarify? What do you mean by "not doing this"? Are the tests skipped correctly with my changes? As far as I can tell, with and without my changes, 20 tests are run on both Python 2 and 3.